### PR TITLE
Slightly modify gearbox names

### PR DESCRIPTION
**Pull request by TheDude53**
Just figured it'd be nice to give the gearbox names a bit more consistency. Let me know what you guys think. Apologies in advance if I come off as a grammar nazi.

### CODE REVIEW

File: source/main/Application.cpp [hunk 1]
```diff
@@ -365,10 +365,10 @@ const char* CONF_VEGET_50PERC       = "50%";
 const char* CONF_VEGET_FULL         = "Full (best looking, slower)";
 
 const char* CONF_GEARBOX_AUTO       = "Automatic shift";
-const char* CONF_GEARBOX_SEMIAUTO   = "Manual shift - Auto clutch";
-const char* CONF_GEARBOX_MANUAL     = "Fully Manual: sequential shift";
+const char* CONF_GEARBOX_SEMIAUTO   = "Manual shift: auto clutch";
```
Comment (only-a-ptr) [inline source/main/Application.cpp:368]: ouch, these can't be changed - they are saved in people's config files. The reason for not using simple mnemonics is historical.

**Review by only-a-ptr — CHANGES_REQUESTED** [review]
Mostly OK except the first commit - remove it please.

**Comment by TheDude53** [comment]
No problem.

**Comment by tritonas00** [comment]
Works fine, just make it one commit.

Are `stick` values meant for controller?

**Comment by only-a-ptr** [comment]
Closing in favor of https://github.com/RigsOfRods/rigs-of-rods/pull/2839

@TheDude53 Thanks for creating this PR but no offense, the commit history is a mess. First off, you should create a separate branch with some sensible name when starting a project. Second, when asked to remove a commit, just do interactive rebase and delete it completely, revert-commits are for things which are already in upstream and rebasing isn't an option. Tip: I personally rely on http://gitextensions.github.io/ GUI, with the commandline it takes extreme concentration to figure out what's happening whereas in the GUI you just see it.

**Comment by TheDude53** [comment]
Thanks for the tips and cleaning this thing up. Happy holidays!

**Pull request closed without merging**